### PR TITLE
Add some variable tests

### PR DESCRIPTION
**Pull request by Lizard-13**

### CODE REVIEW

**Comment by Lizard-13** [comment]
Since I'm here, I would like to discuss some points:
* I had problems launching the tests, the "test" script in the package is launched without the "node" command so I added it, not sure if this is a problem in my system, maybe I need to set a new environment variable or something?
* There was a problem linking the browser launched by karma and the npm tests, I've solved it adding "transports: ['xhr-polling', 'jsonp-polling']" option to karma, but I have no idea how it works so I didn't make the commit.
* I was trying to make a set of variable-copying functions, so you can copy the children from one variable to another, I did it here: [https://github.com/Lizard-13/GD/tree/variable-copy-and-tests](url), but not sure to pull it, since there is a lot of action bloating: Each tipe of variable (scene, global and object) need an action per type, so you get 9 actions!, I was thinking of one action per type with optional parameters to copy from scene, global, object, but you know, it looks weird :/

File: GDJS/tests/tests/variable.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Common tests for gdjs game engine.
+ * See README.md for more information.
+ */
+
+describe('gdjs.variable', function() {
+
+	it('should parse initial values into strings and numbers', function(){
+
+		var int_var = new gdjs.Variable( {value : "526"} );
```
Comment (4ian) [inline GDJS/tests/tests/variable.js]: Could you change the variable notation from snake_case to camelCase (`intVar`...)? Thanks! :)

**Comment by 4ian** [comment]
Thanks for the PR! I made a comment about variables naming, can you update it? I'll merge this once it's done!

For the first problem, I think it's related to Windows. No idea concerning the second problem :/ 
What is the browser and version that you're using with Karma?

Copying variables would be nice, for now the only way to do this is by serializing variables to JSON and then read it back into another.
Not sure what is the best approach though, 9 functions is indeed a lot! We would have just 3 actions if we could have a parameter whose type is an arbitrary expression referencing a variable. But for now expression are only able to return string or numbers :/

**Comment by Lizard-13** [comment]
Fixed!, sorry, python has betrayed me... I think I'm understanding this thing about commits, branches, push :)
About the conversion through JSON, the problem is that those functions works with scene variables only: The expression stays like "ToJSON(MySceneVar)", so no chance to use a global/object variables, and the action to convert a JSON string to variable accept scene variables only too :(

**Comment by 4ian** [comment]
Thanks, merged! :) 
Yes for now there is no simple solution for copying variables I'm afraid :/

**Comment by Lizard-13** [comment]
What do you think about adding support for other variable types in the JSON functions?
jsonconversion
This way you get only 3 actions instead 9, and 3 expressions, and as you can see you can copy one variable into another in a sinlge action, and native is working too :)

**Comment by 4ian** [comment]
Mmm that's indeed something that would be a bit better!
Maybe a long term solution would be to rework actions/conditions so that we can choose which variable (scene, global or one from an object) is used for any variable parameter.
But that would be a good start to allow convert from/to JSON using any variable 👍

**Pull request merged**